ear-production-suite-plugins/plugins/direct_speakers/src/channel_gains_box.hpp

```diff
@@ -42,8 +42,8 @@ class ChannelGainsBox : public Component {
   void updateChannelGainBounds() {
     auto area = getLocalBounds();
     for (auto channelGain : channelGains_) {
-      channelGain->setBounds(area.removeFromLeft(50));
-      area.removeFromLeft(6);
+      channelGain->setBounds(area.removeFromLeft(52));
+      area.removeFromLeft(5);
```

Comment:
Do you think it's worth moving these magic numbers into named constants? I'm not sure exactly what these do from the code (mostly as I'm a bit rubbish with JUCE - my fuzzy brain can't figure out what the difference in appearance is from the two different calls to removeFromLeft, if they had descriptive names it might make that easier)

Reply:
It removes a section from an area and returns it, so the first one takes a 52px wide area for the channelGain component, and the second removes a section to create a margin before the next one. 
(6*52) + (5*5) fills the new width better than (6*50) + (5*6).
Will stick them in constants whilst I'm in there